OpenRA.Mods.Common/Effects/Missile.cs

```diff
@@ -76,6 +76,9 @@ public class MissileInfo : IProjectileInfo
 		[Desc("Vertical rate of turn.")]
 		public readonly int VerticalRateOfTurn = 6;
 
+		[Desc("Gravity applied while in free fall.")]
```

Comment:
I agree with @reaperrr.  This claims to be gravity, not an acceleration vector (which would indeed need to be negative).